Dependencies/BabylonNativeUtils/CMakeLists.txt

```diff
@@ -1,13 +1,25 @@
-set(SOURCES
-    "Include/Babylon/NetworkUtils.h"
-    "Include/Babylon/TicketedCollection.h"
-    "Source/NetworkUtils.cpp")
+if(APPLE)
```

Comment:
nit: I'm a little iffy about the platform-specific guards we have in here. I know they're inevitable in certain cases, but we really ended up with a ton of them before the plugin refactor (many of which we still have), and for me it felt pretty messy. Can we look for alternatives for how to minimize these, at least within scenarios where we have duplicated code, as below?

Reply:
best alternative is to have specific plugin/project per platform IMHO. instead of guards we would have more cmakelists.